docs/deploying.md

```diff
@@ -481,6 +481,25 @@ OpenStackDataPlaneNodeSet has the following status fields:
 | DNSClusterAddresses | |
 | CtlplaneSearchDomain | |
 
+##### OpenStackDataPlaneNodeSet Pods
+
+Pods within OpenStackDataPlaneNodeSet can fail for many reasons.
+Under normal circumstances, these failures may be transient.
```

Comment:
I don't understand this. Transient failures aren't normal circumstances. There shouldn't be any failures. The environment might cause transient or inconsistent failures (due to networking perhaps), but that's not normal.

Reply:
Shouldn't be yes. I'll drop that.